documentation/docs/basics/streams.md

```diff
@@ -1,8 +1,17 @@
+---
+image: /img/integration-services/logo/integration_services.png
+description: IOTA Streams offers auditable encrypted data streams for any industry, ensuring data has not been modified throughout the industry data pipeline.channels.
```

Comment:
```suggestion
description: IOTA Streams offers auditable encrypted data streams for any industry, ensuring data has not been modified throughout the industry data pipeline.channels.
```

Reply:
I'm just not sure if pipeline.channels is formatted like that, my eyes just caught it. If its okay, disregard!